test/mocha/60-performance.js

```diff
@@ -8,7 +8,7 @@ const bedrock = require('bedrock');
 const blsMongodb = require('bedrock-ledger-storage-mongodb');
 const helpers = require('./helpers');
 const mockData = require('./mock.data');
-const uuid = require('uuid/v4');
+const {util: {uuid}} = bedrock;
```

Comment:
Is there a reason why we are using `bedrock.util.uuid` instead of pulling a `uuid` lib?

Reply:
It's available, and it makes sense to use it if it's available.  And it's available because I think it makes sense for `bedrock.util`to offer some primitive helpers.

Reply:
Should we use this pattern in all of our bedrock applications? Using this idea we should do the same thing for frontend applications, right? I don't see any real need for managing different uuid libraries that do the same thing.

Reply:
@gannan08 seems like you should submit an issue somewhere regarding this question.